### ISSUE TITLE
Run tests in parallel

### ISSUE THREAD

**Issue by guidomb** [Quick/Quick#383]
It would be nice to be able run test in parallel. Lets say you want split all your test specs into 4 worker threads. This could help to reduce build times in CI.

Is it possible?

**Comment by modocache** [comment]
Yes, this would be very nice. Currently, Quick is implemented on top of XCTest, which runs all tests serially on the main queue--so parallel execution of tests is not available.

I'm closing this in favor of #385, which will track decoupling Quick from XCTest. Thanks for the question, @guidomb! :clap:
